user.go

```diff
@@ -254,21 +224,16 @@ func NewUserClaimsFromMap(m map[string]interface{}) (*UserClaims, error) {
 		}
 	}
 
-	if len(u.Roles) == 0 {
```

Comment:
@njones , please sync with the latest commit. This one is my "app_metadata" change.